tests/e2e/e2e_routesv2_pod_client_server_test.go

```diff
@@ -0,0 +1,139 @@
+package e2e
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	. "github.com/openservicemesh/osm/tests/framework"
+)
+
+// TODO : remove as a part of routes refactor (#2397). Routes v2 will become the default and this test will no longer be needed
+var _ = OSMDescribe("RoutesV2 Test HTTP traffic from 1 pod client -> 1 pod server",
+	OSMDescribeInfo{
+		Tier:   2,
+		Bucket: 2,
+	},
+	func() {
+		Context("RoutesV2 SimpleClientServer with a Kubernetes Service for the Source: HTTP", func() {
+			testTrafficRoutesV2(true)
+		})
```

Comment:
Don't need to test this combination with every test. Let's remove this as it is already being tested by other e2e tests.

Reply:
fixed, PTAL